src/app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,10 @@
+class TagsController < ApplicationController
+  LIMIT = 30
+
+  def show
+    tag_name = params[:id]
+    @functions = Tag.find_by(name: tag_name).functions
```

Comment:
it's good we have a show method, but of course we need to complete the pagination, that means using Limit and offset and a more button in the view and a js view like all other paginated lists